src/components/NewsFinder/NewsFinder.css

```diff
@@ -16,9 +16,9 @@
 }
 @media (max-width: 590px) {
     .finder__container{
-        margin: 0 16px;
+        margin: 0;
         max-width: calc(100% - 32px);
-        padding: 30px 0 32px;
+        padding: 30px 16px 32px;
     }
 }
```

Comment:
**Might be better**



You can dynamically update the year in `Footer` with `new Date().getFullYear()`